dist/sockjs.js

```diff
@@ -51,7 +51,7 @@ var JSON;JSON||(JSON={}),function(){function str(a,b){var c,d,e,f,g=gap,h,i=b[a]
 //     [*] Including lib/index.js
 // Public object
 var SockJS = (function(){
-              var _document = document;
+              var _document = Pusher.runtime.getDocument();
```

Comment:
This is a built file so the changes would be lost when a release were built.

Right now Pusher knows about SockJS, but not the other way around. It's probably worth trying to come up with an alternative solution.